frontend/Dockerfile.prod

```diff
@@ -1,13 +1,13 @@
 # syntax=docker/dockerfile:1.3-labs
-FROM node:21.0.0 AS builder
+FROM denoland/deno:alpine AS builder
```

Comment:
My only hesitation about alpine is... alpine can be weird in terms of managing and adding packages if we need to do this. In my experience, it seems to make more sense just to use `bookworm` to avoid the problems associated with alpine.

Postgres would of course be an exception to this because you typically don't need to add anything to a postgres container at all. Not sure if this was your thinking here...?